chapters/testing.Rmd

```diff
@@ -983,22 +1004,23 @@ $ git add .travis.yml requirements.txt
 $ git commit -m "Adding requirements"
 ```
 ```text
-[master d96593f] Adding requirements
- 2 files changed, 16 insertions(+), 1 deletion(-)
+[master 4917a19] Adding requirements
+ 2 files changed, 9 insertions(+)
  create mode 100644 requirements.txt
 ```
 ```bash
 $ git push origin master
 ```
 ```text
-Enumerating objects: 4, done.
-Counting objects: 100% (4/4), done.
+Enumerating objects: 6, done.
+Counting objects: 100% (6/6), done.
 Delta compression using up to 4 threads
-Compressing objects: 100% (2/2), done.
-Writing objects: 100% (3/3), 344 bytes | 344.00 KiB/s, done.
-Total 3 (delta 0), reused 0 (delta 0)
+Compressing objects: 100% (3/3), done.
+Writing objects: 100% (4/4), 445 bytes | 74.00 KiB/s, done.
+Total 4 (delta 1), reused 0 (delta 0), pack-reused 0
+remote: Resolving deltas: 100% (1/1), completed with 1 local object.
 To https://github.com/amira-khan/zipf.git
-   1f0590b..71084f7  master -> master
+   46cd2d8..4917a19  master -> master
 ```
 
 Travis CI automatically runs again.
```

Comment:
Sorry if you're already planning to do this, but you'll need to re-create `figures/testing/travis-build-fail.png` and `figures/testing/travis-build-fail.png`, because those are currently screen shots from my GitHub account.

Reply:
Yup, I'm on this!